lib/Target/CBackend/CBackend.cpp

```diff
@@ -4427,7 +4426,7 @@ void CWriter::printIntrinsicDefinition(FunctionType *funT, unsigned Opcode,
 
   if (isa<VectorType>(retT)) {
     // this looks general, but is only actually used for ctpop, ctlz, cttz
-    Type **devecFunParams = (Type **)alloca(sizeof(Type *) * numParams);
+    Type **devecFunParams = (Type **)_alloca(sizeof(Type *) * numParams);
```

Comment:
Uh, what does this have to do with LLVM 13?

Reply:
was a compilation issue it might be from compiling on windows